apps/site/assets/ts/schedule/components/ScheduleFinder.tsx

```diff
@@ -31,13 +31,22 @@ interface State {
 interface SelectContainerProps {
   children: ReactNode;
   error: boolean;
+  handleClick?: Function;
 }
 
 const SelectContainer = ({
   children,
-  error
+  error,
+  handleClick
 }: SelectContainerProps): ReactElement<HTMLElement> => (
-  <div className={`schedule-finder__select-container ${error && "error"}`}>
+  <div
+    className={`schedule-finder__select-container ${error && "error"}`}
```

Comment:
This `error && "error"` returns `false` if there isn't an error, and so there's an extra class "false" which we don't want. Unlikely that we'd ever name a class `false` but worth preventing if a dependency or something did.